autonomy/replay/agent.py

```diff
@@ -102,6 +102,9 @@ def start(
         subprocess.run(  # nosec # pylint: disable=subprocess-run-check
             [*self.aea_cli, "add-key", "ethereum"], env=self.agent_env
         )
+        subprocess.run(  # nosec # pylint: disable=subprocess-run-check
+            [*self.aea_cli, "add-key", "ethereum-flashbots"], env=self.agent_env
+        )
```

Comment:
@angrybayblade I am not sure about this one. I followed the approach above for the ethereum key.